terraform/modules/k8s/monitoring/main.tf

```diff
@@ -1,22 +1,8 @@
 provider "helm" {
-  version         = "0.9.1"
+  version         = "v0.10.0"
   install_tiller  = false
 }
 
-provider "google" {
```

Comment:
Why it was here? @alexander-semenets @karbyshevds

Reply:
It was copypast.